packages/salesforcedx-vscode-lwc/test/vscode-integration/commands/forceLightningLwcPreview.test.ts

```diff
@@ -6,13 +6,14 @@
  */
```

Comment:
Not on you, but at some point soon we are going to need split this into multiple files.

Reply:
Agreed. This file is getting too big.